name: Bug report
description: Create a report to help us improve
labels: bug
body:
  - type: textarea
    id: description
    attributes:
      label: Description of the bug
      description: A clear and concise description of what the bug is.
    validations:
      required: true
  
  - type: textarea
    id: repex
    attributes:
      label: Minimal reproducible example
      description: A code snipped that we can copy&paste to reproduce the bug without having any data
      render: python
      placeholder: |
        import dandelion as ddl
        vdj = ddl.Dandelion(airr_dataframe)

        ddl.do_something_that_doesnt_work(vdj)

  - type: textarea
    id: stacktrace
    attributes:
      label: The error message produced by the code above
      description: Please paste the entire error message
      render: pytb
      placeholder: |
        ---------------------------------------------------------------------------
        Traceback (most recent call last):
          File "<stdin>", line 1, in <module>
          File "/Users/kt16/Documents/Github/dandelion/dandelion/utilities/_core.py", line 49, in __init__
            self.data = load_data(data)
          File "/Users/kt16/Documents/Github/dandelion/dandelion/utilities/_utilities.py", line 564, in load_data
            raise FileNotFoundError(
        FileNotFoundError: Either input is not of <class 'pandas.core.frame.DataFrame'> or file does not exist.
   
  - type: textarea
    id: os
    attributes:
      label: OS information
      description: |
        e.g. MacOS

  - type: textarea
    id: versions
    attributes:
      label: Version information
      description: |
        Please paste the output of `dandelion.logging.print_header()` here.
  
  - type: textarea
    id: context
    attributes:
      label: Additional context
      description: |
        Add any other context about the problem here.
